pages.pt_BR/common/units.md

```diff
@@ -1,8 +1,7 @@
 # units
 
 > Realiza a conversão entre duas unidades de medida.
-> Digitar `search{{text}}` no console retornará uma lista de todas as unidades que contêm `{{text}}`.
```

Comment:
>Typing `search {{text}}` in the console will return a list of all units that contain `{{text}}`.

This could merit its own example 🤔

Reply:
But that's not really a command, since it is used in interactive mode.

Reply:
True, but we still document that type of thing.  Take [`vim`](https://github.com/tldr-pages/tldr/blob/master/pages/common/vim.md) or [`less`](https://github.com/tldr-pages/tldr/blob/master/pages/common/less.md), for example.

Reply:
I added it to the english page. Not sure what we should do about the translations however.

Reply:
Leave them.  Due to the change in the example counts, https://lukwebsforge.github.io/tldri18n/ will mark the translations as out-of-date.